px-dropdown-content.html

```diff
@@ -407,6 +460,52 @@
           return computedItemsArr;
         }
       }
+    },
+    /**
+     * This function filters items in the dropdown
+     * and show only those which math the searchTerm
+     */
+    _computeFilter: function(searchTerm) {
+      if (!this.searchMode || !searchTerm.length) {
+        // set filter to null to disable filtering
+        return null;
+      } else {
+        // return a filter function for the current search string
+        searchTerm = searchTerm.toLowerCase();
+        return function(entry) {
```

Comment:
`filter` property of `dom-repeat` template needs function in return statement to work properly with dynamic values (like when we want to use some input search with custom values). 
I was following [Polymer docs](https://www.polymer-project.org/1.0/docs/devguide/templates#dynamic-sort-and-filter-changes)